# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mp4/MP4Metadata.cpp
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 dom/media/mp4/MP4Metadata.cpp
--- a/dom/media/mp4/MP4Metadata.cpp	Fri Jan 27 19:06:16 2023 +0000
+++ b/dom/media/mp4/MP4Metadata.cpp	Fri Jan 27 21:11:42 2023 +0200
@@ -32,8 +32,7 @@
 
 size_t IndiceWrapper::Length() const { return mIndice.length; }
 
-bool IndiceWrapper::GetIndice(size_t aIndex,
-                              MP4SampleIndex::Indice& aIndice) const {
+bool IndiceWrapper::GetIndice(size_t aIndex, Index::Indice& aIndice) const {
   if (aIndex >= mIndice.length) {
     MOZ_LOG(gMP4MetadataLog, LogLevel::Error, ("Index overflow in indice"));
     return false;